File: modul-326/bomberman-modul-326/unity/Bomberman_Returns/Assets/Models/ReceiveMessages/GameOver.cs
using System.Collections;
using System.Collections.Generic;

public class GameOver : Message
{

    private string winnerName;
    private string[] highscoreList;

    public GameOver(string winnerName, string[] highscoreList)
    {
        this.winnerName = winnerName;
        this.highscoreList = highscoreList;
    }

    public string WinnerName { get => winnerName; }
    public string[] HighscoreList { get => highscoreList; }
}

File: modul-326/bomberman-modul-326/src/main/java/ch/gibb/bomberman/protocol/server2client/GameOver.java
package ch.gibb.bomberman.protocol.server2client;

import ch.gibb.bomberman.network.Message;

public class GameOver implements Message {
  private String winnerName;
  private String[] highscoreList;

  public GameOver(String winnerName, String[] highscoreList) {
    this.winnerName = winnerName;
    this.highscoreList = highscoreList;
  }

  public String getWinnerName() {
    return winnerName;
  }

  public String[] getHighscoreList() {
    return highscoreList;
  }
}
